src/app/forms/templates/general/hgv-trl-body.template.ts

```diff
@@ -0,0 +1,53 @@
+import { ValidatorNames } from '@forms/models/validators.enum';
+import { FormNode, FormNodeTypes } from '../../services/dynamic-form.types';
+
+export const hgvAndTrlBodyTemplate: FormNode = {
+  name: 'bodySection',
+  label: 'Body',
+  type: FormNodeTypes.GROUP,
+  children: [
+    {
+      name: 'make',
+      label: 'Body make',
+      value: '',
+      type: FormNodeTypes.CONTROL,
+      validators: [{ name: ValidatorNames.MaxLength, args: 30 }, { name: ValidatorNames.Required }]
+    },
+    {
+      name: 'model',
+      label: 'Body model',
+      value: '',
+      type: FormNodeTypes.CONTROL,
+      validators: [{ name: ValidatorNames.MaxLength, args: 30 }, { name: ValidatorNames.Required }]
+    },
+    {
+      name: 'bodyType',
+      label: 'Body type',
+      value: '',
+      type: FormNodeTypes.GROUP,
+      children: [
+        {
+          name: 'description',
+          label: 'Body type',
```

Comment:
Are we not having a dropdown of just standard values for body type instead of free text? I think the backend expects certain input for this field (screenshot from the BE API code).
![image](https://user-images.githubusercontent.com/92087051/198977464-137f73e9-4368-4bd5-aa77-f408605ed4be.png)

Reply:
Oh wait we're using the multi options service now so the form options aren't shown in the templates right?

Reply:
All of the values shown on your image match our `BodyTypeDescription` enum. So, it should be fine.